sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -153,8 +152,7 @@ void _ottf_main(void) {
 
   // Clear reset reason register.
   dif_rstmgr_t rstmgr;
-  CHECK_DIF_OK(dif_rstmgr_init(
-      mmio_region_from_addr(TOP_EARLGREY_RSTMGR_AON_BASE_ADDR), &rstmgr));
+  CHECK_DIF_OK(dif_rstmgr_init_from_dt(kDtRstmgrAon, &rstmgr));
```

Comment:
nit: `kDtRstmgrAon` is a top-specific name, and this could be especially problematic for OTTF integration. Not every integration is going to have a rstmgr with this sort of AON / non-AON power domain split. We might need to think about how init might need to split off an integration layer specific to each top. (normal OS / operating environment stuff)